pkg/filter/dt/filter_mysql_query.go

```diff
@@ -48,20 +51,22 @@ func (f *_mysqlFilter) processBeforeQueryDelete(ctx context.Context, conn *drive
 		return nil
 	}
 	executor := exec.NewQueryDeleteExecutor(conn, deleteStmt)
-	bi, err := executor.BeforeImage(ctx)
+	bi, err := executor.BeforeImage(newCtx)
 	if err != nil {
 		return err
 	}
-	if !proto.WithVariable(ctx, beforeImage, bi) {
+	if !proto.WithVariable(newCtx, beforeImage, bi) {
 		return errors.New("set before image failed")
 	}
 	return nil
 }
 
 func (f *_mysqlFilter) processBeforeQueryUpdate(ctx context.Context, conn *driver.BackendConnection, updateStmt *ast.UpdateStmt) error {
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_before_query_update")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.BeforeImage`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "fetch_before_image")
```